test/deprecated/error-msg-removed.good

```diff
@@ -1 +1 @@
-error-msg-removed.chpl:1: error: Error constructor accepting a string is no longer available
+error-msg-removed.chpl:1: error: Error initializer accepting a string is no longer available
```

Comment:
I think this error could just be removed, since it got deprecated back in November.

Reply:
Similar to the BlockCyclic comment, I thought I'd leave that for a separate issue.